src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts

```diff
@@ -25,15 +25,15 @@ import {Conditionals} from '../../shared/edit/conditional-fn';
 
 import {ZakenVerdelenDialogComponent} from '../zaken-verdelen-dialog/zaken-verdelen-dialog.component';
 import {ZakenVrijgevenDialogComponent} from '../zaken-vrijgeven-dialog/zaken-vrijgeven-dialog.component';
-import {ZakenWerkvoorraadDatasource} from './zaken-werkvorraad-datasource';
+import {ZakenWerkvoorraadDatasource} from './zaken-werkvoorraad-datasource';
 
 @Component({
     selector: 'zac-zaken-werkvoorraad',
```

Comment:
Haal deze selector ook maar weg. Wordt nergens gebruikt. Als een selector er niet is weet je als lezer van de code ook meteen dat deze niet gebruikt wordt en dat het component op een andere manier wordt geinstantieerd